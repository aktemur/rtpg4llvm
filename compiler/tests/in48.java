/*stmt*/
int __restrict__ x;
int *__restrict__ x;
int __restrict__ x[3];
int *__restrict__ x[3];


