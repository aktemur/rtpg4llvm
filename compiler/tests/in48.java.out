mkSeqStmt(4, new VarDecl(new IntType(), "x", true),
      new VarDecl(new PtrType(new IntType()), "x", true),
      new VarDecl(new ArrayType(new IntType(),3), "x", true),
      new VarDecl(new ArrayType(new PtrType(new IntType()),3), "x", true))
